src/Umbraco.Core/StringExtensions.cs

```diff
@@ -1478,5 +1478,24 @@ public static string ToSafeFileName(this string text, IShortStringHelper shortSt
         {
             return shortStringHelper.CleanStringForSafeFileName(text, culture);
         }
+
+        /// <summary>
+        /// Validates a string matches a time.
+        /// </summary>
+        /// <param name="input">Time representation (in Hmm form, e.g. 330, 2200)</param>
+        /// <returns></returns>
+        public static bool IsValidTimeFormat(this string input)
+        {
+            if (string.IsNullOrEmpty(input) || input.Length < 3 || input.Length > 4)
+            {
+                return false;
+            }
+
+            var inputWithSeparator = input.Length == 3
+                ? $"{input.Substring(0, 1)}:{input.Substring(1)}"
+                : $"{input.Substring(0, 2)}:{input.Substring(2)}";
+
+            return TimeSpan.TryParse(inputWithSeparator, out var _);
```

Comment:
I feel like this should simply be. Augmenting the input is too black box.

``` c#
return TimeSpan.TryParse(input, out var _);
```

Reply:
Should probably have flagged the thinking here too... the issue I found was that the currently configuration value is expected to be a time formatted like this: `Hmm` - so `330` or `1640`.  To validate though, it seems that `TimeSpan.TryParse("2500")` returns true, not false.

So I've added in the hour/minute separator, making `2500` to `25:00`, and when you try parse that, it returns false as you'd expect.

Hence in trying to retain the existing configuration format and be able to validate it as being a valid time, I've done this.

Reply:
I think a breaking change is warranted then as mentioned above by @bergmania  

Incidentally I saw a tweet triggered by bad use of primitives to configure time spans in System.Web.
https://twitter.com/khellang/status/1321461708301475842

Reply:
Yes, agreed, if we're minorly breaking one of the config values we just as well tidy this one up too - so it now accepts a time format of `H:mm` - e.g. `12:00`, which this validation works with as expected.